### PR TITLE
DAO.sol readability changes

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

File: DAO.sol [hunk 1]
```diff
@@ -204,7 +220,6 @@ contract DAO is DAOInterface, Token, Crowdfunding {
         _voteID = p.votes.length++;
         p.votes[_voteID] = Vote({inSupport: _supportsProposal, voter: msg.sender});
         p.voted[msg.sender] = true;
-        p.numberOfVotes = _voteID + 1;
```
Comment (CJentzsch) [inline DAO.sol]: why do you want to remove that?

Comment (LefterisJP) [inline DAO.sol]: What purpose does it serve? If someone wants to query the number of votes can't they simply query the length of the `p.votes` array?

Comment (CJentzsch) [inline DAO.sol]: in theory, yes. Need to check of the autogenerated Mist UI can do that. I think that was the reason for that.

Comment (LefterisJP) [inline DAO.sol]: all right sure, let's check it. In the meantime then let me reduce this PR to only the readability changes.

**Comment by CJentzsch** [comment]
sorry, after merging the other PRs there seems to be a merge conflict. Please rebase.

**Comment by LefterisJP** [comment]
rebased

File: DAO.sol [hunk 2]
```diff
@@ -105,11 +105,15 @@ contract DAOInterface {
     address public serviceProvider;
     address[] public allowedRecipients;
 
-    mapping (address => uint) public rewardRights;  //only used for splits, give DAOs without a balance the privilige to access there share of the rewards
-    uint public totalRewardRights;
+
+    //only used for splits, give DAOs without a balance the privilige to access their share of the rewards
+    mapping (address => uint) public rewardRights;
+    uint public accumulatedRewardRights;
```
Comment (CJentzsch) [inline DAO.sol]: it needs to be `totalRewardRights` (you removed it a couple of lines above) . This was changed in another PR.

**Pull request merged**